runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileWrapperBuilder.java

```diff
@@ -30,7 +30,7 @@
     @Deprecated // TODO: Add replacement method that conforms to style
     @SuppressWarnings("checkstyle:abbreviation") // Due to deprecation
     public static <T> T getWrapper(Class<T> type, CorfuRuntime rt,
-                                   UUID streamID, Object[] args,
+                                   UUID streamID, String streamName, Object[] args,
```

Comment:
I think its fine to drop the UUID and just pass the stream name.

Reply:
Sounds good